src/thing/get.ts

```diff
@@ -281,6 +281,43 @@ export function getStringWithLocaleAll(
   return matchingQuads.map((quad) => quad.object.value);
 }
 
+/**
+ * Retrieves all string literals for the specified property from the specified
+ * Thing.
```

Comment:
This links to the Thing interface.

```suggestion
 * [[Thing]].
```